tree/ntuple/v7/inc/ROOT/RNTupleOptions.hxx

```diff
@@ -44,22 +45,30 @@ All page sink classes need to support the common options.
 */
 // clang-format on
 class RNTupleWriteOptions {
-  int fCompression{RCompressionSetting::EDefaults::kUseAnalysis};
-  ENTupleContainerFormat fContainerFormat{ENTupleContainerFormat::kTFile};
-  bool fUseBufferedWrite = true;
+   int fCompression{RCompressionSetting::EDefaults::kUseAnalysis};
+   ENTupleContainerFormat fContainerFormat{ENTupleContainerFormat::kTFile};
+   NTupleSize_t fNClusterEntries = 64000;
+   NTupleSize_t fNElementsPerPage = 10000;
+   bool fUseBufferedWrite = true;
 
 public:
-  int GetCompression() const { return fCompression; }
-  void SetCompression(int val) { fCompression = val; }
-  void SetCompression(RCompressionSetting::EAlgorithm algorithm, int compressionLevel) {
-    fCompression = CompressionSettings(algorithm, compressionLevel);
-  }
+   int GetCompression() const { return fCompression; }
+   void SetCompression(int val) { fCompression = val; }
+   void SetCompression(RCompressionSetting::EAlgorithm algorithm, int compressionLevel) {
+     fCompression = CompressionSettings(algorithm, compressionLevel);
+   }
 
-  ENTupleContainerFormat GetContainerFormat() const { return fContainerFormat; }
-  void SetContainerFormat(ENTupleContainerFormat val) { fContainerFormat = val; }
+   ENTupleContainerFormat GetContainerFormat() const { return fContainerFormat; }
+   void SetContainerFormat(ENTupleContainerFormat val) { fContainerFormat = val; }
 
-  bool GetUseBufferedWrite() const { return fUseBufferedWrite; }
-  void SetUseBufferedWrite(bool val) { fUseBufferedWrite = val; }
+   NTupleSize_t GetNClusterEntries() const { return fNClusterEntries; }
```

Comment:
Let's rename to `GetNEntriesPerCluster` so that it is consistent with `GetNElementsPerPage` (also for the setter and the data member)

Reply:
Thank you, done in b9654b4